### PR TITLE
Categories' sidebar

### PR DESCRIPTION
**Pull request by patrykpiston**

### CODE REVIEW

File: src/pages/Category.vue [hunk 1]
```diff
@@ -61,7 +61,8 @@ export default {
       category: {},
       filters: { // filters should be set by category, and should be synchronized with magento
         color: ['red', 'blue'],
```
Comment (pkarw) [inline src/pages/Category.vue]: Sizes and colors must be mapped to some database-related values. For example "color" in DB is: 169, 180 etc. The same with size.

Can you provide data model for that? separate key / value or label / caption - you name it :) - pairs would be great @patrykpiston

Comment (pkarw) [reply src/pages/Category.vue]: I'll add a mechanism to fill out these dictionaries from database

Comment (pkarw) [reply src/pages/Category.vue]: This is key to merge this into master :-)

Comment (patrykpiston) [reply src/pages/Category.vue]: Sure, once I get some time today / tomorrow I'll get back to it and come up with something :) @pkarw

File: src/pages/Category.vue [hunk 2]
```diff
@@ -60,8 +60,14 @@ export default {
       products: {},
       category: {},
       filters: { // filters should be set by category, and should be synchronized with magento
-        color: ['red', 'blue'],
-        size: ['XS', 'S', 'M', 'L', 'XL'],
+        color: [{'id': 165, 'label': 'red'}, {'id': 166, 'label': 'blue'}],
+        size: [
```
Comment (patrykpiston) [inline src/pages/Category.vue]: I've put the key-value pairs here as these are just test data. Or should they be placed in resources/models folder while leaving these fields empty to be filled later?

Comment (pkarw) [reply src/pages/Category.vue]: Seems just fine for me. I'll add some db-logic to load the appropriate attribute options in here. It's fine for me

**Pull request merged**